docs/azure_jumpstart_ag/contoso_supermarket/freezer_monitor/_index.md

```diff
@@ -4,4 +4,264 @@ weight: 100
+
+  ![Visual Studio Code showing the Simulator logs](./img/vscode_simulator_logs.png)
+
+#### MQTT Broker
+
+The MQTT Broker is a container running Mosquitto in each AKS Edge Essentials cluster like the simulator. It receives the data from the simulator and sends it to the Azure IoT Hub. It also makes the data available for a third service, MQTT2Prometheus, which we'll discuss in a moment.
+
+To see data being received by the MQTT Broker
+
+Assuming you have completed the steps to view the MQTT Simulator logs above:
+
+- Expand __Chicago__ > __Workloads__ > __Pods__ then right-click on the 'sensor-monitor-simulator-xxx' pod and select 'Logs'
+
+  ![Visual Studio Code showing the Logs menu](./img/vscode_broker_logs_menu.png)
+- In the Logs view click __Run__ to see the logs
+
+  ![Visual Studio Code showing the Logs window](./img/vscode_broker_logs_run.png)
+- This won't show you the values being received or forwarded, but it will show you the connections from the 2 simulated freezer devices to the broker, as well as the connections from the broker to Azure IoT Hub for each freezer device. Finally, it shows the connection from __sensor-monitor-mqtt2prom__ which subscribes to the freezer data on the broker and makes it available to Prometheus, but more on that a bit later.
+
+  ![Visual Studio Code showing the Simulator logs](./img/vscode_broker_logs.png)
+
+#### Azure IoT Hub
+
+From the MQTT Broker the data is sent to Azure IoT Hub, which is a managed service, hosted in the cloud, that acts as a central message hub for bi-directional communication between your IoT application and the devices it manages. You can use IoT Hub to build IoT solutions with reliable and secure communications between millions of IoT devices and a cloud-hosted solution backend. You can connect virtually any device to IoT Hub.
+
+To see whether data is being received by Azure IoT Hub for your devices
+
+From your local machine:
+
+- Open Resource Groups in the Azure Portal - [https://portal.azure.com](https://ms.portal.azure.com/#browse/resourcegroups)
+- Click the new resource group you created for __Jumpstart Agora__
+- Click __Ag-IotHub-xxxxx__ to open the IoT Hub
+
+  ![Azure Portal showing the IoT Hub](./img/azure_portal_iot_hub.png)
+- Click __Queries__ in the left navigation menu
+
+  ![Azure Portal showing the IoT Hub Queries](./img/azure_portal_iot_hub_queries.png)
+- Click __Run query__
+
+  ![Azure Portal showing the IoT Hub Run query button](./img/azure_portal_iot_hub_queries_run.png)
+  
+- Scroll down to where you see __"deviceId": "Freezer-1-Chicago"__
+- Review the __"connectionState"__ and __"lastActivityTime"__ values to see if the device is connected and sending data
+
+  ![Azure Portal showing the IoT Hub Run query results](./img/azure_portal_iot_hub_queries_results.png)
+- Repeat the last 2 steps above for __"Freezer-2-Chicago"__
+
+#### Azure Data Explorer (ADX)
```

Comment:
Do we need this section? Reference to the first occurrence of ADX name should be enough for the user to visit ADX service documentation page and learn more.

Reply:
Removed